sidecar.sh

```diff
@@ -43,35 +42,56 @@ populate_notebook() {
 function populate() {
     log "querying manifest service at $GEN3_ENDPOINT/manifests/"
     MANIFESTS=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/manifests/")
-    while [ -z "$MANIFESTS" ]; do
-        log "Unable to get manifests from '$GEN3_ENDPOINT/manifests/'"
-        log $MANIFESTS
+    log "querying manifest service at $GEN3_ENDPOINT/metadata/"
+    METADATA=$(curl -s -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://$GEN3_ENDPOINT/metadata/")
+
+    while [ -z "$MANIFESTS" ] || [ -z "$METADATA" ]; do
```

Comment:
I think this should be a `&&`, otherwise it will stuck forever if there is no metadata, which is likely

Reply:
I agree, we don't want the loop to stop unless both conditions are satisfied, I've addressed this in the latest commit